pages/project/read/views/components/manage-access.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Snippet, Link } from '@asl/components';
+import Section from './section';
+import Collaborators from '../../../components/collaborators';
+
+export default function ManageAccess({ model }) {
+  const canUpdate = useSelector(state => state.static.canUpdate);
+  if (!canUpdate) {
+    return null;
+  }
+
+  return (
+    <Section
+      title={<Snippet>manageAccess.title</Snippet>}
+      content={<Snippet>manageAccess.content</Snippet>}
+    >
+      <p>
+        <Link page="project.addUser" label={<Snippet>manageAccess.action</Snippet>} />
+      </p>
+      {
+        !!model.collaborators.length && <Collaborators collaborators={model.collaborators} />
```

Comment:
I'd be tempted to handle this in the component, but there's not necessarily any harm in having it here too.